src/notation/internal/notationconfiguration.cpp

```diff
@@ -299,19 +310,42 @@ async::Notification NotationConfiguration::foregroundChanged() const
     return m_foregroundChanged;
 }
 
+bool NotationConfiguration::scoreInversionEnabled() const
+{
+    return settings()->value(INVERT_SCORE_COLOR).toBool();
+}
+
+void NotationConfiguration::setScoreInversionEnabled(bool value)
+{
+    settings()->setSharedValue(INVERT_SCORE_COLOR, Val(value));
+}
+
+async::Notification NotationConfiguration::scoreInversionChanged() const
+{
+    return m_scoreInversionChanged;
+}
+
 io::path NotationConfiguration::wallpapersDefaultDirPath() const
 {
     return globalConfiguration()->appDataPath() + "/wallpapers";
 }
 
 QColor NotationConfiguration::borderColor() const
 {
-    return QColor(0, 0, 0, 102);
+    if (uiConfiguration()->isHighContrast()) {
+        return QColorConstants::White;
+    } else {
+        return QColor(0, 0, 0, 102);
```

Comment:
```c++
return QColor(102, 102, 102, 0);
```

Reply:
Also fixed

Reply:
> Wouldn't your suggested color (102, 102, 102, 0) be fully transparent?

Oops yes! I just swapped the values (i.e. did the reverse of [this](https://github.com/musescore/MuseScore/pull/8886#discussion_r703296369)) but of course the alpha always needs to be 255 to remain full opacity.

> Perhaps this should be left as is? as i assume this acts just as a "shadow" around the score-page?

Yes, if it is really just a shadow then it needs to use alpha transparency, so you can ignore my comment! I thought this might be a dark colour used for the score background or something.